package/kedro_viz/server.py

```diff
@@ -113,6 +124,42 @@ def run_viz(port=None, line=None) -> None:
     display(HTML(wrapper))
 
 
+def _load_from_file(load_file: str) -> dict:
+    global data  # pylint: disable=global-statement,invalid-name
+    data = json.loads(Path(load_file).read_text())
+    for key in ["nodes", "edges", "tags"]:
+        if key not in data:
+            raise KedroCliError(
+                "Invalid file, top level key '{}' not found.".format(key)
+            )
+    return data
+
+
+def _get_pipeline_from_context(context, pipeline_name):
+    if match(kedro.__version__, ">=0.15.2"):
+        return context._get_pipeline(  # pylint: disable=protected-access
+            name=pipeline_name
+        )
+    # Kedro 0.15.0 or 0.15.1
+    if pipeline_name:
+        raise KedroCliError(ERROR_PIPELINE_FLAG_NOT_SUPPORTED)
+    return context.pipeline
+
+
+def _get_pipeline_catalog_from_kedro14(env, pipeline_name):
+    if pipeline_name:
```

Comment:
I think this check can be handled outside of this function, the argument doesn't really belong here.